_layouts/base.html

```diff
@@ -0,0 +1,136 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <!-- Google Tag Manager --
```

Comment:
You should re-enable the tag manager tags. Use the same one as the rest of the site.

Reply:
Extracted them into includable fragments and used those